.github/CODEOWNERS

```diff
@@ -67,9 +67,12 @@
 /CONTRIBUTORS @seanlip
 /LICENSE @seanlip
 /NOTICE @seanlip
-/core/templates/pages/about-page/ @seanlip
-/core/templates/pages/privacy-page/ @seanlip
-/core/templates/pages/terms-page/ @seanlip
+/core/templates/pages/about-page/*.html @seanlip
```

Comment:
Could you make about.html assigned to me and everything else in /about-page assigned to you (no specific file extension), in case new files get added in the future? Ditto for others.